.github/workflows/ci.yml

```diff
@@ -87,7 +87,7 @@ jobs:
 
       - name: Run unit tests with idna_convert 0.5.1
         if: ${{ matrix.with == 'idna-convert-v051' }}
-        run: composer test --bootstrap=tests/bootstrap-with-idna-convert-051.php
+        run: bin/phpunit --bootstrap=tests/bootstrap-with-idna-convert-051.php
```

Comment:
The proper syntax is `composer test -- --bootstrap=tests/bootstrap-with-idna-convert-051.php`

The `--` will stop composer from interpreting arguments as flags and let them pass through.